tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -196,22 +211,30 @@ def test_delete_registered_model(self):
         self.store.delete_registered_model(name=name)
 
         # cannot get model
-        with self.assertRaises(MlflowException) as exception_context:
+        with self.assertRaisesRegex(
+            MlflowException, r"Registered Model with name={} not found".format(name)
+        ) as exception_context:
             self.store.get_registered_model(name=name)
         assert exception_context.exception.error_code == ErrorCode.Name(RESOURCE_DOES_NOT_EXIST)
 
         # cannot update a delete model
-        with self.assertRaises(MlflowException) as exception_context:
+        with self.assertRaisesRegex(
+            MlflowException, r"Registered Model with name={} not found".format(name)
+        ) as exception_context:
             self.store.update_registered_model(name=name, description="deleted")
         assert exception_context.exception.error_code == ErrorCode.Name(RESOURCE_DOES_NOT_EXIST)
 
         # cannot delete it again
-        with self.assertRaises(MlflowException) as exception_context:
+        with self.assertRaisesRegex(
+            MlflowException, r"Registered Model with name={} not found".format(name)
```

Comment:
```suggestion
            MlflowException, fr"Registered Model with name={name} not found"
```

Can we use f-string instead of `format`?

Reply:
Sure. I will fix it.
I just followed the conventional code base, so if it is better to replace all the `format` (including ones irrelevant to this issue) with f-string, I will conduct it as well.

Reply:
We're actually using both. I think we can just replace the ones relevant to this PR.

Reply:
I understand. Thanks for the review. I will work on it!